spring-client-common/src/main/java/io/camunda/commons/http/HttpClient.java

```diff
@@ -17,7 +18,8 @@ public interface HttpClient {
 
   <T> T get(Class<T> clazz, String id);
 
-  void get(String resource);
+  <T> String getXml(Class<T> clazz, Long key);
+
+  <T, V, W, U> T post(Class<T> clazz, Class<V> vj, TypeToken<W> typeToken, U body);
```

Comment:
Could the parameter be a bit more explicit please? I do not understand what `vj` should be and what the typeToken is good for.

Reply:
My bad. I will make it more explicit. Was committing things yesterday and didn't realize i left some test names out there.

Reply:
Addressed in https://github.com/camunda-community-hub/spring-zeebe/pull/510/commits/45441794dbc1d7a37c1d24fb2cf1ed00aaa41e11